modules/astcenc/SCsub

```diff
@@ -40,6 +40,23 @@ env_astcenc.Prepend(CPPPATH=[thirdparty_dir])
 
 env_thirdparty = env_astcenc.Clone()
 env_thirdparty.disable_warnings()
+
+# Compiler options to ensure invariant build.
+# Sync with upstream `Source/cmake_core.cmake` options for ASTCENC_INVARIANCE.
+if env.msvc:
+    # FIXME: MSVC compiler version detection isn't well implemented yet.
+    pass
+#   if env.compiler_version >= (19, 30, 0):
+#       env_thirdparty.AppendUnique(CCFLAGS=["/fp:precise"])
+#   else:
+#       env_thirdparty.AppendUnique(CCFLAGS=["/fp:strict"])
```

Comment:
Will look into this later, but if someone can test on MSVC and confirm that astc encoding still seems to behave ok, I think this can be merged already.